scenes/Ships/Ship_controls.gd

```diff
@@ -84,9 +85,17 @@ func handle_landing_request():
 	PlayerState.selection = closest_planet
 	PlayerState.selection.get_node("Selection").visible = true
 	
-	print (speed_check)
+	print ("Speed = ",speed_check)
 	if speed_check > 250:
```

Comment:
I think we should require that they are moving slower. A value of `50` here I think would be good.